### PR TITLE
[WIP] Added Dockerfile and docker-compose.yml

### PR DESCRIPTION
**Pull request by clintonb**
Docker can be used to get the service, and its dependencies, up and running with minimal effort.

### CODE REVIEW

**Comment by clintonb** [comment]
This is a work-in-progress. Let me know if you *would not* like this contribution. I won't be offended, as I mostly did this work to better understand how all of this fits together...only to learn I need data scrapers from https://github.com/unitedstates/congress.

**Comment by joejordan** [comment]
After several days spent trying to get this API properly configured, I think a Docker solution is a great idea. Any progress since your initial release?

**Comment by clintonb** [comment]
@joejordan I have not been working on this PR since opening it. Now that I have better Docker knowledge, I see some places where the Dockerfile can be greatly improved, if you want to pick up where I left off.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.2
+
+RUN apt-get update
```
Comment (clintonb) [inline Dockerfile]: Combine this with the calls to `apt-get install`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Install pip
+RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python
+RUN pip --version
+
+# Setup source/working directory
+ADD . /usr/src/app
+WORKDIR /usr/src/app
+
+# Install dependencies
+RUN bundle install
```
Comment (clintonb) [inline Dockerfile]: `COPY` the requirements files, and install the requirements. `ADD` the code last.

Comment (clintonb) [inline Dockerfile]: Also, use a `.dockerignore` file to keep the Docker build context small.

**Pull request closed without merging**